docker/app/install-composer.sh

```diff
@@ -4,7 +4,7 @@
 # Origin: https://github.com/composer/composer/blob/master/doc/faqs/how-to-install-composer-programmatically.md
 # Licence: MIT
 
-EXPECTED_SIGNATURE=$(wget -q -O - https://composer.github.io/installer.sig)
```

Comment:
It reduces the complexity of the build process and the maintenance. As pointed out by Jordi, there is no point using the old method when using PHP 7+. And we're using PHP 7+.